### DISCUSSION TITLE
Next build missing files deploying on IIS

### DISCUSSION THREAD

**Discussion by jmacbhc** [vercel/next.js#72435] [Help]
### Link to the code that reproduces this issue

https://github.com/dipneupane/NextJs-IISDeployment

### To Reproduce

1. run next build
2. following steps to deploy on IIS: [https://medium.com/@dipneupane/deploying-nextjs-application-on-windows-iis-server-cabdc22bccf8](url)
3. Done.

### Current vs. Expected behavior

Missing image files on ".next" folder, I expect the page to load all resources. 

![clipboard-image](https://github.com/vercel/next.js/assets/114681459/0d2bebdc-d34a-48a2-b63d-03e66340b1a4)

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 10 Pro
Binaries:
  Node: 18.16.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 13.5.4
  eslint-config-next: 13.2.4
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.0.2
Next.js Config:
  output: standalone
```

### Which area(s) are affected? (Select all that apply)

Standalone mode (output: "standalone")

### Additional context

I'm trying to deploy my application on IIS, using output: "standalone".

**Comment by lextm** [comment]
With HttpPlatformHandler, you just need a few simple steps,

https://stackoverflow.com/a/77823437/11182

**Comment by franferns** [comment]
Hi @lextm , Thanks for the above **HttpPlatformHandler** solution it looks good. I have an issue with correctly accessing the page from the **IIS** set up . Below is how we have configured: 

I have a site hosted in my local iis which looks something like this (see below).  It is an ASP.NET WebForms site 

![image](https://github.com/vercel/next.js/assets/779253/b7cad8cd-9b51-4f5f-809f-380b13dd6fa0)

and the web.config in this configured with these values 

![image](https://github.com/vercel/next.js/assets/779253/77989bd1-4537-400a-bfb7-6bcf25b7ff22)

So from the iis hosted site on the user accessing the url  http://localhost/AppWithNode1/SimpleErrorView should show the nextjs site.

The **NextJS** is of version (14.1.0) 

![image](https://github.com/vercel/next.js/assets/779253/2e8acc7b-a2d5-409e-b6d7-ef2e47da1331)

I have a folder **SimpleErrorView** with a **page.tsx** and some content in it. So when I run the `npm run dev` command and run the http://localhost:3000/SimpleErrorView its shows the page.

![image](https://github.com/vercel/next.js/assets/779253/97598a5a-023f-40af-80ff-cdc37ac86351)


Now, I run the `npm run build` which then builds the **.next** and copy these **node_modules/package.json & .next** to the location where the site is hosted IIS.

Upon accessing the site from iis http://localhost/AppWithNode1/SimpleErrorView it does not show the contents shown when the  http://localhost:3000/SimpleErrorView link is used, instead it shows the **NextJs** 404 page. 

![image](https://github.com/vercel/next.js/assets/779253/1df0ce66-aaa6-4473-bded-30c44e9004d2)

So clearly it runs the **NextJs** site from the IIS just does not point to the correct page. How do I get this to point to the correct page ?

I have tried with using the **basePath** and setting a value to **AppWithNode1** the name of hosted site , it still shows the 404 NextJs error. 

![image](https://github.com/vercel/next.js/assets/779253/b47d233a-94b4-49bb-9de5-9ca65fd43dbe)

Anything I can do to fix this ?

Thanks in advance

**Comment by samcx** [comment]
Hi everyone—

This is likely an issue with the configuration with IIS. One thing Next.js does in Standalone mode (to be minimal) is **not** copy over the static files, you have to do that manually → https://nextjs.org/docs/app/api-reference/next-config-js/output#automatically-copying-traced-files.

The discussions in this issue may provide guidance! → https://github.com/vercel/next.js/issues/49283
